invenio_app_ils/config.py

```diff
@@ -582,7 +588,9 @@ def _(x):
             #     terms=dict(field="internal_location.name"),
             # ),
             circulation_status=dict(
-                terms=dict(field="circulation_status.state"),
+                terms=dict(field="circulation_status.state",
```

Comment:
Match the style of everything else in the file:
```
terms=dict(
    field="circulation_status.state",
    missing="N/A",
),
```

Reply:
this is only if we want to filter with no circulation state - not applicable in the rest of the cases